sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/PostProcessor.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob;
+
+import com.azure.storage.blob.models.StorageErrorException;
+import com.azure.storage.common.Utility;
+import reactor.core.publisher.Mono;
+
+final class PostProcessor {
+    static <T> Mono<T> postProcessResponse(Mono<T> response) {
+        return Utility.postProcessResponse(response, (errorResponse) ->
```

Comment:
Why is this error wrapping kept specific to blobs?

Reply:
This is kept specific to blobs due to each of Blobs, Files, and Queues having their own StorageErrorException class that is generated from the swagger. Each of those use their own specific set of StorageErrorCode. Luckily that is the only logic completely specific to a given SDK and why this class and method exists.